graphql.go

```diff
@@ -377,24 +424,8 @@ func (c *Client) runWithPostFields(ctx context.Context, req *Request, resp inter
 	}
 	c.logf(">> headers: %v", r.Header)
 	r = r.WithContext(ctx)
-	res, err := c.sendRequest(r)
-	if err != nil {
-		return err
-	}
-	defer res.Body.Close()
-	var buf bytes.Buffer
-	if _, err := io.Copy(&buf, res.Body); err != nil {
-		return errors.Wrap(err, "reading body")
-	}
-	c.logf("<< %s", buf.String())
```

Comment:
can we keep this log line?

Reply:
If we are not concerned with memories to print this log line yes!

Reply:
actually I'll just print this log line with the buf I already had. No worries on additional memories